test/functional/page_objects/common_page.js

```diff
@@ -270,6 +270,19 @@ export function CommonPageProvider({ getService, getPageObjects, getPageObject }
     async isChromeVisible() {
       return await testSubjects.exists('kibanaChrome');
     }
+
+    async isTopNavVisible() {
+      return await testSubjects.exists('top-nav');
+    }
+
+    async waitForTopNavToBeVisible() {
```

Comment:
We're only using `waitForTopNavToBeVisible` at the moment, which is going to retry the `isTopNavVisible` call multiple times based on the `timeouts.try` config setting, but the `testSubjects.exists` call has it's own timeout as well.

If we don't have a need for the `isTopNavVisible` method directly, it seems like `waitForTopNavToBeVisible` could be simplified down to:
```
    async waitForTopNavToBeVisible(someNewTimeout) {
      return await testSubjects.exists('top-nav', someNewTimeout);
    }
```
and like we don't need `isTopNavVisible`.